### ISSUE TITLE
kexec fatal error C1083 while running "npm install"

### ISSUE THREAD

**Issue by op1ekun** [systemjs/systemjs#337]
Hi,

I'm getting the following error while trying to `npm install` the latest `master` branch:

```
c:\dev\github\systemjs\node_modules\6to5\node_modules\kexec>node "c:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" configure build
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  kexec.cc
..\src\kexec.cc(12): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory [c:\dev\github\systemjs\node_modules\6to5\node_modules\kexec\build\kexec.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `msbuild` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (c:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:810:12)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command "node" "c:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "configure" "build"
gyp ERR! cwd c:\dev\github\systemjs\node_modules\6to5\node_modules\kexec
gyp ERR! node -v v0.10.30
gyp ERR! node-gyp -v v0.13.1
gyp ERR! not ok
npm WARN optional dep failed, continuing kexec@1.0.0
```

Thanks

**Comment by guybedford** [comment]
This seems like a 6to5 install issue.

This should be resolved with https://github.com/ModuleLoader/es6-module-loader/issues/297.

Release to follow soon.

**Comment by guybedford** [comment]
This should be resolved now in 0.13.0.
